hw/latest/registers/src/i3ccsr.rs

```diff
@@ -87,7 +88,7 @@ impl<TMmio: ureg::Mmio> RegisterBlock<TMmio> {
         }
     }
     #[inline(always)]
-    pub fn i3_c_ec(&self) -> I3cEcBlock<&TMmio> {
+    pub fn i3c_ec(&self) -> I3cEcBlock<&TMmio> {
```

Comment:
The sec_fw_recovery_if is still at 0 instead of i3c_ec + 0 offset.

Reply:
Fixed!